training.py

```diff
@@ -165,7 +166,7 @@ def progress(num_steps, metrics):
         group=args.group_name,
         name=args.exp_name,
         config=vars(args),
-        mode="online" if args.log_wandb else "disabled",
+        mode=mode="online" if args.log_wandb else "disabled",
```

Comment:
I think that's a typo

Reply:
Oops, yeah, fixed. I usually run jobs with mode="offline" and reverted the change incorrectly after testing. Test results should still be fine though.